src/resource.js

```diff
@@ -22,29 +34,35 @@ module.exports = function (settings, method) {
     // create new Observable resource
     const options = getOptions(settings, this._rx, params.rx);
     const source = getSource(this[method].bind(this), arguments);
-    const stream = source.concat(
-      source.exhaustMap(data => {
+    const stream = source.pipe(
+      concatMap(data => {
         // Filter only data with the same id
-        const filter = current => current[options.idField] === data[options.idField];
+        const filterFn = current => current[options.idField] === data[options.idField];
         // `removed` events get special treatment
-        const filteredRemoves = this.removed$.filter(filter);
+        const filteredRemoves$ = this.removed$.pipe(filter(filterFn));
```

Comment:
No deal breaker, personally I'm not a big fan of hungarian notation style flags (here the `$`) in variable names.

Reply:
Yeah, sorry, I shouldn't mess with your naming conventions.
Observables are the only place where I use to do it, anyway. We can revert those.